main.js

```diff
@@ -0,0 +1,28 @@
+var phrase = 'grumpy wizards make toxic brew for the evil queen'
+var array = []
+var $phraseContainer = document.getElementsByTagName('div')[1]
+for(var c = 0; c < phrase.length; c++) {
+  var obj = {}
+  obj[c] = phrase[c]
```

Comment:
You want these objects to be homogenous.

```js
var character = {
  letter: phrase[c]
}
```

That way you don't need a weird `for...in` loop later.